homeassistant/auth/__init__.py

```diff
@@ -479,6 +510,38 @@ def async_remove_refresh_token(self, refresh_token: models.RefreshToken) -> None
         for revoke_callback in callbacks:
             revoke_callback()
 
+    @callback
+    def _async_remove_expired_refresh_tokens(self, _: datetime | None = None) -> None:
+        """Remove expired refresh tokens."""
+        now = time.time()
+        for token in self._store.async_get_refresh_tokens()[:]:
```

Comment:
If we need to copy the list I suggest we use `.copy()` as that's more clear what the intention is.

But it doesn't look like we need to make a copy since we already create a new list in `store.async_get_refresh_tokens`. I suggest we just remove the full slice operation.